web/src/styles/sections/nav-bar.scss

```
@@ -28,14 +28,31 @@ $icon-offset: 45px;
   z-index: 5;
 }
 
+.Mui-focusVisible.nav-menu-item-container {
+  background-color: $transparent;
+}
+
+.Mui-focusVisible.nav-menu-item-container.bg-primary-extra-light-mui-selected-override {
+  background-color: $primary-extra-light;
+}
+
+.Mui-focusVisible.profile-non-selected {
+  background-color: #fbfdf9;
+}
+
 .nav-menu-item-container {
   min-height: 47px;
 }
 
-.profile-default-background-color {
+.profile-non-selected {
   background-color: #fbfdf9;
 }
 
+.profile-non-selected:hover {
+  background-color: $primary-extra-light;
+  font-weight: 700;
+}
+
```

With Sass you can nest related pseudo selectors
```suggestion
.profile-non-selected {
  background-color: #fbfdf9;
  &:hover {
    background-color: $primary-extra-light;
    font-weight: 700;
  }
}
```